api/hypershift/v1alpha1/hostedcluster_types.go

```diff
@@ -1574,6 +1574,26 @@ type AzurePlatformSpec struct {
 	SubnetID          string `json:"subnetID"`
 	SubscriptionID    string `json:"subscriptionID"`
 	SecurityGroupID   string `json:"securityGroupID"`
+
+	// payloadArchitecture specifies if a single CPU architecture payload or a multi-arch CPU payload is being used for
+	// the HostedCluster and HostedControlPlane. When this field is set to Multi, additional validation checks will be
+	// performed on the release image of the HostedCluster to confirm it is a multi-arch image.
+	//
+	// The default for this field is Multi since this allows both amd64 and arm64 NodePools to be created from day 0
```

Comment:
You've marked the field as required, so it cannot have a default?

Reply:
I don't understand. A field cannot be required and have a default value?

Reply:
Defaulting is applied after validation, so marking it required, would reject the request when the field is not populated, but marking it optional, would allow the object to be persisted, and then the default would be applied when being read